RLTSquare/EmailSender/Model/EmailSend.php

```diff
@@ -0,0 +1,86 @@
+<?php
+declare(strict_types=1);
+
+namespace RLTSquare\EmailSender\Model;
+
+use Magento\Email\Model\BackendTemplate;
+use Magento\Framework\App\Area;
+use Magento\Framework\DataObject;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Exception\MailException;
+use Magento\Framework\Mail\Template\FactoryInterface;
+use Magento\Framework\Mail\Template\TransportBuilder;
+use Magento\Store\Model\Store;
+use RLTSquare\EmailSender\Api\TemplateRepositoryInterface;
+
+/**
+ *package for setting template content and user emails
+ */
+class EmailSend
+{
+    /**
+     * @var BackendTemplate
+     */
+    protected BackendTemplate $emailTemplate;
+    /**
+     * @var TransportBuilder
+     */
+    protected TransportBuilder $transportBuilder;
+
+    /**
+     * @var FactoryInterface
+     */
+    protected FactoryInterface $factory;
+
+    /**
+     * @var TemplateRepositoryInterface
+     */
+    protected TemplateRepositoryInterface $templateRepository;
+
+    /**
+     * @param TransportBuilder $transportBuilder
+     * @param BackendTemplate $emailTemplate
+     * @param FactoryInterface $factory
+     * @param TemplateRepositoryInterface $templateRepository
+     */
+    public function __construct(
+        TransportBuilder            $transportBuilder,
+        BackendTemplate             $emailTemplate,
+        FactoryInterface            $factory,
+        TemplateRepositoryInterface $templateRepository
+    ) {
+        $this->factory = $factory;
+        $this->transportBuilder = $transportBuilder;
+        $this->emailTemplate = $emailTemplate;
+        $this->templateRepository = $templateRepository;
+    }
+
+    /**
+     * @throws MailException
+     * @throws LocalizedException
+     */
+    public function execute(): void
+    {
+        $template_id = $this->templateRepository->getById('email_template');
```

Comment:
Overlooked logic - the logic here is faulty. The requirement was to email template that user can edit and change from admin. That implies use of system.xml, with field specific for juggling email_template, which in turn gets set to default value of coded email template using config.xml.

```
               <field id="email_template" translate="label comment" type="select" sortOrder="30" showInDefault="1" showInWebsite="1" showInStore="1" canRestore="1">
                    <label>Email Template</label>
                    <comment>Email template chosen based on theme fallback when "Default" option is selected.</comment>
                    <source_model>Magento\Config\Model\Config\Source\Email\Template</source_model>
                </field>
```

Then, a helper method can be used for reading the template

```
    public function getEmailTemplate(): string
    {
        return (string)$this->scopeConfig->getValue(
            self::XML_PATH_EMAIL_TEMPLATE,
            ScopeInterface::SCOPE_STORE
        );
    }
```

```
            $transport = $this->transportBuilder
                ->setTemplateIdentifier($this->helper->getEmailTemplate())
```

this way we ensure either default meail template is used (through cdefaults in config.xml) or a reassigned one is used ... all through that one helper call